index.html

```diff
@@ -22,6 +22,32 @@
       font-family: -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Helvetica, Arial, "Apple Color Emoji", "Segoe UI Emoji", "Segoe UI Symbol", sans-serif;
```

Comment:
Can we update the body font-size to `1rem` or `medium` so it's relative to the user settings.
(I know that's from a previous commit, but i missed it before)